### PR TITLE
feat: Combine cost and tag policies comment output

### PR DESCRIPTION
**Pull request by tim775**

### CODE REVIEW

**Review by aliscott — APPROVED** [review]
Nice!

File: internal/output/markdown.go [hunk 1]
```diff
@@ -161,6 +161,9 @@ func ToMarkdown(out Root, opts Options, markdownOpts MarkdownOptions) ([]byte, e
 		"showProject": func(p Project) bool {
 			return showProject(p, opts, false)
 		},
+		"cloudURL": func() string {
```
[the remark this answers is not in the document]
Comment (tim775) [reply internal/output/markdown.go:164]: I couldn't see another way to pass the value from the toplevel template to the sub-template.

[the remark this answers is not in the document]
Comment (aliscott) [reply internal/output/markdown.go:164]: 👍

File: cmd/infracost/testdata/comment_bitbucket_with_tag_policy_checks/comment_bitbucket_with_tag_policy_checks.golden [hunk 2]
```diff
@@ -0,0 +1,135 @@
...
+
+Comment not posted to Bitbucket (--dry-run was specified)
+
+Err:
+Error: Tag policy check failed:
+
+  Failing Tag Policy: This should show as a failure
+    module.hosted_cloud_pricing_api.aws_s3_bucket.cloud_pricing_api_db_data in project(s) my-dev-project, my-other-project-with-same-violations
+    - should have mandatory tags: "Environment", "Team"
+    - should have a valid value for "Team" tag: "Ops", "Frontend team", "Backend team"
```
Comment (vdmgolub) [inline cmd/infracost/testdata/comment_bitbucket_with_tag_policy_checks/comment_bitbucket_with_tag_policy_checks.golden:104]: @tim775 Do we want to keep this copy as is or update it to follow the same "Missing mandatory tags"/"Invalid value..." messages?

**Pull request merged**